embedded-graphics/src/image/image16bpp.rs

```diff
@@ -0,0 +1,208 @@
+//! 16 bits per pixel images. Every two bytes define the color for each pixel.
+//! 
+//! You can convert an image to 16BPP for inclusion with `include_bytes!()` doing the following
+//! 
+//! Converting the Image to 16bit BMP (GIMP can do this), then running 
+//! ```bash
+//! tail -c $bytes image.bmp > image.raw
+//! ```
+//! where $bytes is `w * h * 2` will remove the BMP header leaving the raw data
```

Comment:
The grammar is a bit confusing here. What do you mean by `will remove the BMP header leaving the raw data`?